lib/Conversion/CGGIToJaxite/CGGIToJaxite.td

```diff
@@ -0,0 +1,15 @@
+#ifndef LIB_CONVERSION_CGGITOJAXITE_CGGITOJAXITE_TD_
+#define LIB_CONVERSION_CGGITOJAXITE_CGGITOJAXITE_TD_
+
+include "mlir/Pass/PassBase.td"
+
+def CGGIToJaxite : Pass<"cggi-to-jaxite"> {
+  let summary = "Lower `cggi` to `jaxite` dialect.";
+  let dependentDialects = [
+    "mlir::arith::ArithDialect",
+    "mlir::heir::lwe::LWEDialect",
```

Comment:
not sure you need this one either since we're guaranteed to have loaded the LWE dialect to parse the input IR

Reply:
Removed.